src/components/manifold-subscription-create/manifold-subscription-create.tsx

```diff
@@ -296,7 +318,7 @@ export class ManifoldSubscriptionCreate {
           </p>
           {/* TODO restyle success state when designs are available */}
           {this.setupIntentStatus === 'succeeded' && (
-            <Message type="success">You've been subscribed!</Message>
+            <Message type="success">Setup intent completed!</Message>
```

Comment:
Is this visible to users? My only thought is that I have no idea what setup intent is hahaha and it might be confusing to users

Reply:
Yeah, this will be removed fairly soon. I just changed it so it made sense for now since subscribing doesn't actually happen at this point.